### PR TITLE
add testscene-run action

### PR DESCRIPTION
**Pull request by pipipipipi43**
Execute the corresponding pipeline according to the scene name and parameter configuration, and then wait for the execution to complete

issue:
https://terminus-org.app.terminus.io/erda/workBench/projects/387/issues/requirement?id=70716&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&type=requirement

### CODE REVIEW

**Comment by sfwn** [comment]
add a logo for this action.

File: actions/testscene-run/1.0/spec.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: testscene-run
+version: '1.0'
+type: action
+displayName: 自动化测试场景执行
+category: deploy_management
+desc: 选择指定场景进行测试
+public: true
+supportedVersions:
```
Comment (sfwn) [inline actions/testscene-run/1.0/spec.yml:8]: useErdaVersions

Comment (sfwn) [reply actions/testscene-run/1.0/spec.yml:8]: useProtocol

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by pipipipipi43** [comment]
related erda pr: https://github.com/erda-project/erda/pull/1218

**Review by sfwn — APPROVED** [review]

**Pull request merged**